app/models/character_race.rb

```diff
@@ -0,0 +1,4 @@
+class CharacterRace < ApplicationRecord
+  belongs_to :character
+  belongs_to :race
+end
```

Comment:
Will have to find a way to use ActiveRecord to make this required for a character to be created